src/services/OrganizationService.ts

```diff
@@ -54,10 +69,36 @@ type UpdateOrganizationRequestPayload =
 type OrganizationRolesServerResponse =
   paths[typeof ORGANIZATION_ROLES_ENDPOINT]['get']['responses'][200]['content']['application/json'];
 
+const httpAcceleratorOrganizations = HttpService.root(ACCELERATOR_ORGANIZATIONS_ENDPOINT);
 const httpOrganizations = HttpService.root(ORGANIZATIONS_ENDPOINT);
 const httpOrganization = HttpService.root(ORGANIZATION_ENDPOINT);
 const httpOrganizationRoles = HttpService.root(ORGANIZATION_ROLES_ENDPOINT);
 
+/** get accelerator organizations */
+const getAcceleratorOrganizations = async (): Promise<AcceleratorOrganizationsResponse> => {
+  const response: AcceleratorOrganizationsResponse = await httpAcceleratorOrganizations.get<
+    AcceleratorOrganizationsServerResponse,
+    AcceleratorOrganizationsData
+  >(
+    {
+      params: {
+        includeParticipants: 'true',
+      },
+    },
+    (data) => ({ organizations: data?.organizations ?? [] })
+  );
+
+  if (!response.requestSucceeded) {
+    if (response.statusCode === 401) {
+      response.error = 'NotAuthenticated';
+    } else {
+      response.error = 'GenericError';
+    }
+  }
+
+  return response;
+};
+
```

Comment:
Same comment regarding location, it seemed like the best place but I can move it no problem.

Reply:
@joshbuchea the accelerator orgs code already exists, this was needed for participant creation as well.
See PR https://github.com/terraware/terraware-web/pull/2389/files for types/service/redux